PolyEngine/Games/SandboxRendering/Src/RenderingSandbox.cpp

```diff
@@ -100,38 +123,117 @@ void RenderingSandbox::Deinit()
 	}
 };
 
-void GameMainSystem::GameUpdate(Poly::World* world)
+float RenderingSandbox::Random()
+{
+	return static_cast <float> (rand()) / static_cast <float> (RAND_MAX);
+}
+
+float RenderingSandbox::Random(float min, float max)
 {
-	double time = world->GetWorldComponent<TimeWorldComponent>()->GetGameplayTime();
-	double deltaTime = TimeSystem::GetTimerDeltaTime(world, Poly::eEngineTimer::GAMEPLAY);
+	float rnd = Random();
+	return Lerp(min, max, rnd);
+}
 
-	// int UseCashetes = 0;
-	// float ColorTempLuminancePreservation = 0.75f;
-	float Distortion	= 0.5f;
-	float Bloom			= 0.4f;
-	float ColorTempValue = 9500.0f; // 6500.0 from [1000.0, 40000.0]
-	float Saturation	= 0.5f;
-	float Grain			= 0.5f;
-	float Stripes		= 0.5f;
-	float Vignette		= 0.5f;
-	float Brightness	= 0.1f;
-	float Contrast		= 1.5f;
+void RenderingSandbox::AddDirectionalLights() 
```

Comment:
This method is not used anymore, either remove it or move the implementation from `Init()` here.

Reply:
done